src/Tizen.NUI.Scene3D/src/public/ModelMotion/MotionIndex/MotionTransformIndex.cs

```diff
@@ -0,0 +1,196 @@
+            /// </summary>
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            PositionX,
+
+            /// <summary>
+            /// The y position of ModelNode. MotionValue should be float.
+            /// </summary>
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            PositionY,
+
+            /// <summary>
+            /// The z position of ModelNode. MotionValue should be float.
+            /// </summary>
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            PositionZ,
+
+            /// <summary>
+            /// The orientation of ModelNode. MotionValue should be Rotation.
+            /// </summary>
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            Orientation,
+
+            /// <summary>
+            /// The scale of ModelNode. MotionValue should be Vector3.
+            /// </summary>
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            Scale,
+
+            /// <summary>
+            /// The x scale of ModelNode. MotionValue should be float.
+            /// </summary>
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            ScaleX,
+
+            /// <summary>
+            /// The y scale of ModelNode. MotionValue should be float.
+            /// </summary>
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            ScaleY,
+
+            /// <summary>
+            /// The z scale of ModelNode. MotionValue should be float.
+            /// </summary>
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            ScaleZ,
+        }
+
+        /// <summary>
+        /// Create an initialized blend shape index.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public MotionTransformIndex() : this(Interop.MotionIndex.MotionTransformIndexNew(), true)
+        {
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        /// <summary>
+        /// Create an initialized blend shape index with input node id, and transform type.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public MotionTransformIndex(PropertyKey modelNodeId, TransformTypes transformType) : this(Interop.MotionIndex.MotionTransformIndexNew(PropertyKey.getCPtr(modelNodeId), (int)transformType), true)
+        {
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        /// <summary>
+        /// Copy constructor.
+        /// </summary>
+        /// <param name="motionTransformIndex">Source object to copy.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public MotionTransformIndex(BlendShapeIndex motionTransformIndex) : this(Interop.MotionIndex.NewMotionTransformIndex(MotionTransformIndex.getCPtr(motionTransformIndex)), true)
```

Comment:
```suggestion
        public MotionTransformIndex(MotionTransformIndex motionTransformIndex) : this(Interop.MotionIndex.NewMotionTransformIndex(MotionTransformIndex.getCPtr(motionTransformIndex)), true)
```
Please change this  typo error